.gitignore

```diff
@@ -85,3 +85,6 @@ _releasenotes
 # Used during parts of the build process. Files _should_ get cleaned up automatically.
 # This is also a good location for any temporary manfiests used during development
 tmp
+
+# asdf
```

Comment:
asdf? :)

Reply:
That's actually the name: https://asdf-vm.com/

Reply:
It's useful to switch tool versions between projects. Adding it to avoid adding my local tool configuration.

Reply:
Ah okay. Fine for me to add it, can just add slightly more wording around it so it becomes more obvious that it's not just a typo? :D

Reply:
No problem. Added a little more descriptive comment